packages/base-contract/src/index.ts

```diff
@@ -184,7 +184,7 @@ export class BaseContract {
         }
         return rawEncoded;
     }
-    public async evmExecAsync(input: Buffer): Promise<string> {
+    protected async _evmExecAsync(input: Buffer): Promise<string> {
```

Comment:
Is the reason for this just to avoid having documentation for it generated into the markdown files?  Or is there some other reason as well?

Reply:
Partly to avoid having docs generated, partly because it's not used anywhere else, so follows the principle of minimum exposed surfaces. 

It doesn't make a whole lotta sense to call it in an external context. It only works for functions defined in the Solidity and available via bytecode; and those should all be accessed by calling `functionName.callAsync` or similar. I have a feeling we should also avoid bypassing those to directly pass in callData even for unit tests because that would be an easy way to create holes in our testing.